packages/transaction-controller/src/gas-flows/DefaultGasFeeFlow.test.ts

```diff
@@ -70,19 +83,17 @@ const FEE_MARKET_EXPECTED_RESULT: GasFeeEstimates = {
 };
 
 // Converted to Hex and multiplied by 1 billion.
-const LEGACY_EXPECTED_RESULT: GasFeeEstimates = {
-  low: {
-    maxFeePerGas: '0x3b9aca00',
-    maxPriorityFeePerGas: '0x3b9aca00',
-  },
-  medium: {
-    maxFeePerGas: '0xb2d05e00',
-    maxPriorityFeePerGas: '0xb2d05e00',
-  },
-  high: {
-    maxFeePerGas: '0x12a05f200',
-    maxPriorityFeePerGas: '0x12a05f200',
-  },
+const LEGACY_EXPECTED_RESULT: LegacyGasFeeEstimates = {
+  type: GasFeeEstimateType.Legacy,
+  low: '0x3b9aca00',
+  medium: '0xb2d05e00',
+  high: '0x12a05f200',
+};
+
+// Converted to Hex and multiplied by 1 billion.
```

Comment:
[nit] We could see if a pattern like this could work:
```js
const gasPriceEstimate = 3;
const GAS_PRICE_RESPONSE_MOCK = {
  gasEstimateType: GAS_ESTIMATE_TYPES.ETH_GASPRICE,
  gasFeeEstimates: {
    gasPrice: String(gasPriceEstimate),
  },
} as GasFeeState;

// [...]

const LEGACY_EXPECTED_RESULT: LegacyGasFeeEstimates = {
  type: GasFeeEstimateType.Legacy,
  low: decimalToHex(1000000000),
  medium: decimalToHex(gasPriceEstimate * 10 ** 9),
  high: decimalToHex(5000000000),
};
```
This way we avoid having to decode hex manually and can read what is happening directly in the test code without needing to trust/keep the comment up to date